moveit_planners/ompl/ompl_interface/src/detail/ompl_constraints.cpp

```diff
@@ -310,6 +310,36 @@ void EqualityPositionConstraint::jacobian(const Eigen::Ref<const Eigen::VectorXd
   }
 }
 
+/******************************************
+ * Orientation constraints
+ * ****************************************/
+void OrientationConstraint::parseConstraintMsg(const moveit_msgs::msg::Constraints& constraints)
```

Comment:
I'd prefer to see `bounds_`, `target_orientation_`, and `link_name_` passed into this function so it's clear what's being modified.

```suggestion
void OrientationConstraint::parseConstraintMsg(const moveit_msgs::msg::Constraints& constraints, Bounds& bounds, ...)
```

Reply:
I'll change this to at least pass around `bounds` and `link_name`, but as below, orientation is tricky. Once we decide on what we can do for orientation, I can apply that here.

Reply:
I'm also not sure about this because `EqualityConstraint` doesn't actually modify `bounds_`, but if we add it as an argument, we run into the same inheritance issue. We can pass it in anyway, and just not use it, but I think that defeats the whole point about being clear on what's being modified.